### DISCUSSION TITLE
catenary program internal_forces distribution

### DISCUSSION THREAD

**Discussion by cuicw** [GazzolaLab/PyElastica#348] [Q&A]
I tried to run the catenary program, and I compared it with the distribution of the data obtained by the lumped mass method. It was found that the value of internal _ forces seemed to be unreasonable. As shown in the following figure, the value of the endpoint seems to be much larger than the value of the intermediate node.
![image](https://github.com/GazzolaLab/PyElastica/assets/154817929/43689b62-c8d6-41f5-ab16-8634d695ae42)

**Comment by armantekinalp** [comment]
@Ali-7800 can you take a look this.

**Comment by Ali-7800** [comment]
@cuicw can you describe how you used the lumped mass method?

**Reply by cuicw** [reply]
I used the Orcaflex software and then simulated the static results of the catenary structure, as shown in the following figure
![image](https://github.com/GazzolaLab/PyElastica/assets/154817929/9876053f-bdb4-45a0-8b25-dde9720f26bb)

**Comment by Ali-7800** [comment]
Looking at your plot it seems your catenary is 300m long while ours is only 1m long. How are you exactly comparing the two results? Are you matching the same properties between the two programs such as density, elastic modulus, radius, etc? or are you doing some sort of scaling?

**Reply by cuicw** [reply]
Yes, I try to build a big scale catenary model , But I don 't know the difference between internal _ force and internal _ stress, their array length is different. The endpoint of internal _ force has the above problem, but internal _ stress is not, I want to clarify their definition

**Reply by Ali-7800** [reply]
You can read this [paper](https://www.cosseratrods.org/publications/pubs/2018_RSOS.pdf), it explains the definitions in more detail.
